docs/rules/no-multi-assign.md

```diff
@@ -3,7 +3,11 @@
 Chaining the assignment of variables can lead to unexpected results and be difficult to read.
 
 ```js
-a = b = c = d;
+(function() {
+    const foo = bar = 0; // do you mean `foo = bar == 0`?
```

Comment:
Can we write these as complete sentences for a more consistent tone?

Reply:
Like this?
```
// Do you mean `foo = bar == 0`?
// This will not fail, `bar` is not constant.
// This will output 1, `bar` is not scoped.
```

Reply:
@kaicataldo Friendly ping: Could you check out the reply to your comment when you have time? Thanks!

Reply:
Thanks for the ping. I was thinking something along the lines of:

```
// Did you mean `foo = bar == 0`?
// This will not fail since `bar` is not constant.
// This will output 1 since `bar` is not scoped.
```

Reply:
Done.